matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/events/model/EventType.kt

```diff
@@ -70,6 +70,9 @@ object EventType {
     const val STATE_ROOM_ENCRYPTION = "m.room.encryption"
     const val STATE_ROOM_SERVER_ACL = "m.room.server_acl"
 
+    // This type is for local purposes, it should never be processed by the server
+    const val LOCAL_STATE_ROOM_THIRD_PARTY_INVITE = "local.room.third_party_invite"
```

Comment:
Could be internal, as well as `LocalRoomThirdPartyInviteContent` (and maybe move this class to the internal package?)

Reply:
https://github.com/vector-im/element-android/pull/6127/commits/2be2a057959f2c9588d2f8b8cc3af91a6abd277a